Airwallex/Core/Sources/Internal/Extensions/Codable+Utils.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Codable+Utils.swift
+//  Core
+//
+//  Created by Tony He (CTR) on 2024/7/22.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+public extension Encodable {
+    func toDictionary() -> [String: Any]? {
```

Comment:
Encodable object can be parsed into Json object, is it still necessary to convert to dictionary?

Reply:
This is to support some of our OC model which use NSDictionary in its parameters function.